public/app/features/manage-dashboards/components/SnapshotListTable.tsx

```diff
@@ -3,6 +3,7 @@ import useAsync from 'react-use/lib/useAsync';
 
 import { getBackendSrv, config } from '@grafana/runtime';
 import { ConfirmModal, Button, LinkButton } from '@grafana/ui';
+import {t,Trans} from 'app/core/internationalization'
```

Comment:
this is causing a lint error. if you install our precommit hooks via `make lefthook-install` it will run prettier automatically when you commit 👍